contents/stats.md

```diff
@@ -0,0 +1,445 @@
+    Options(fig; filename=label, caption, label)
+    """
+sco(s)
+```
+
+In this chapter, we will cover only descriptive statistics.
+Inferential statistics is an important and fundamental component of applied sciences, but its scope is too broad. Let's begin on simple ways to summarize our data with central tendencies.
+
+## Central Tendencies Measures {#sec:stats_central}
+
+The most basic way of using descriptive statistics is to summarize data by a measure of **central tendency**.
+
+### Mean {#sec:stats_central_mean}
+
+The most common central tendency measure is the mean.
+**The mean, also known as average, is the sum of all measurements divided by the number of observations** and we generally denote it as "x bar":
+
+$$ \bar{x} = \frac{1}{n} \sum^n_{i=1} x_i = \frac{x_1 + x_2 + \cdots + x_n}{n}, $$ {#eq:mean}
+
+where $\bar{x}$ is the sample mean of the variable $\mathbf{x} = x_1, \cdots, x_n$.
+Often, we see the mean denoted with the **Greek letter $\mu$**, for example $\bar{x} = \mu_x$.
+Additionally, the mean is also known as the **expectation** which is represented by the operator $\operatorname{E}$, thus the mean of a variable $x$ becomes a $\operatorname{E}(x)$.
+So, bear in mind that you might find different notations for the mean.
+
+The mean can be used from the `mean` function from Julia's standard library `Statistics` module:
+
+```julia
+using Statistics: mean
+```
+
+And we can apply the mean to different groups in our data like we did in @sec:groupby_combine.
+For example, we have the `all_grades` `DataFrame`:
+
+```jl
+# inside stats content # hide
+sco("all_grades()"; process=without_caption_label)
+```
+
+Let us add more grades to our students so that we have more numbers to calculate central tendencies:
+
+```jl
+sco("more_grades()"; process=without_caption_label)
+```
+
+```jl
+# inside stats content # hide
+s = """
+    gdf = groupby(more_grades(), :name)
+    combine(gdf, :grade => mean)
+    """
+sco(s; process=without_caption_label)
+```
+
+### Median {#sec:stats_central_median}
+
+We will see that the mean is highly sensitive to outliers and can sometimes misguide us, specially when we are dealing with long-tailed (non-normal distributions) (more in @sec:stats_dist_normal).
+That is why sometimes we are interested in also the **median** which is the **middle value that separates the higher half from the lower half of the data**.
+Intuitively, the median tells us the value of the data's 50\% percentile.
+One example is when we are analyzing income.
+The median is the upper limit that we expect that half of the observations earn.
+So, if the median income is U$ 80,000, we expect that half of our observations earn between the minimum value and U$ 80,000.
+The mathematical formula for the median is:
+
+$$ \operatorname{median}(\mathbf{x}) = \frac{x_{\lfloor (\# x+1) \div 2 \rfloor} + x_{\lceil (\# x+1) \div 2 \rceil}}{2}, $$ {#eq:median}
+
+where:
+
+- $\mathbf{x} = x_1, \cdots, x_n$ is an ordered list of numbers
```

Comment:
```suggestion
- $\mathbf{x} = x_1, \cdots, x_n$ is an ordered vector of numbers
- $\mathbf{x}_i$ is the element in vector $\mathbf{x}$ at position $i$
```
I propose vector because it's more consistent with Julia's terminology.